### PR TITLE
Dev mariateo opensource

### PR DESCRIPTION
**Pull request by MariaTeo**
Finished the Open source section

### CODE REVIEW

File: src/_component/about-me/about-me.scss [hunk 1]
```diff
@@ -20,16 +20,7 @@
 }
 
 .pill--blue {
-  height: 40px;
-  margin: 0 auto;
-  color: white;
-  border:0;
```
Comment (biggicode) [inline src/_component/about-me/about-me.scss:26]: space

**Review by biggicode — APPROVED** [review]
Well done 💯

**Review by screpyy96 — APPROVED** [review]
💯

File: src/_component/open-source/open-source.scss [hunk 2]
```diff
@@ -0,0 +1,294 @@
...
+
+// .eleGit {
+//   display: grid;
+//   grid-template-columns: auto 1fr auto;
+//  }
+
+// .cacaunu {
+//   color: #808080;
+//   font-size: 14px;
+//   font-weight: medium;
```
Comment (BogdanNegura) [inline src/_component/open-source/open-source.scss:22]: you can delete all this commented code if you won't need it anymore.

**Review by BogdanNegura — APPROVED** [review]
👍

File: src/_component/open-source/open-source.scss (hunk 2 above)
Comment (MariaTeo) [reply src/_component/open-source/open-source.scss:22]: Oh god haha. I deleted that, I do not know why it is still visible 🙈

**Pull request merged**